### ISSUE TITLE
Some fonts don't comply with the GPL

### ISSUE THREAD

**Issue by Hiradur** [RigsOfRods/rigs-of-rods#450]
Some fonts are not entirely free and thus don't comply with the GPL.
Namely:
```
bluebold.ttf
bluecond.ttf
bluehigh.ttf
Cyberbit.tff
digital-7.mono.ttf
micross.ttf
```

Some of these fonts may not be used at all, this has to be investigated. However, the Cyberbit font seems to play a major role for localization.

Among licensing issues this also means that RoR can't be packaged for the Debian "main" software repository right now since these fonts break with the Debian Free Software Guidelines ( http://www.debian.org/social_contract ).

**Comment by Hiradur** [comment]
From a quick fulltext search I think it'd be safe to remove these fonts as I don't think they are used anywhere:
```
bluebold.ttf
bluecond.ttf
bluehigh.ttf
micross.ttf
```

``` digital-7.mono.ttf ``` appears to be used by dashboards and ```Cyberbit.tff``` is used in many places so we need a replacement font with a free/libre license for these.
We already have VeraMono so that may be a candidate. Needs investigation. Replacing Cyberbit.ttf won't be so easy since it's used for localization and as such we need a font with a large character set.

**Comment by Hiradur** [comment]
After replacing ```Cyberbit.tff``` it's important to check if and how code ranges* for localization are affected.

* more about code ranges here: http://www.rigsofrods.com/wiki/pages/Language_Translation#How_it_works

**Comment by only-a-ptr** [comment]
@Hiradur Thanks for looking into this.

"BlueHighway" sounds familiar. Either it used to be part of Ogre sample media, or it used to be part of CEGUI. I remember seeing the name in Ogre logs when doing tutorials.
Anyway, we're not the only ones having this concern: http://forum.freegamedev.net/viewtopic.php?p=59035&sid=caf6519b31a7ae55ebad8e7245dc90aa#p59035

**Comment by Hiradur** [comment]
@only-a-ptr Yes, they were used for OGRE samples and have since been removed from OGRE due to licensing issues: https://ogre3d.atlassian.net/browse/OGRE-76

**Comment by Hiradur** [comment]
The Vera fonts can't replace Cyberbit because they contain only latin characters. 
The DeJaVu fonts are often proposed but don't contain the chinese characters for example (http://dejavu.sourceforge.net/samples/DejaVuSans.pdf )

The biggest free font is GNU Unifont:
http://unifoundry.com/unifont.html

it's ~12mb big which matches the file size of Cyberbit.tff. A character set this large may seem like overkill but it's definetely the most future-proof and which is important considering the code range hassle mentioned above.

**Comment by Hiradur** [comment]
I removed these 4 fonts:

```
bluebold.ttf
bluecond.ttf
bluehigh.ttf
micross.ttf
```

and discovered that bluebold.ttf is used for the gear indicator...

**Comment by AnotherFoxGuy** [comment]
Replacement for ```digital-7.mono.ttf```?
https://github.com/ctrlcctrlv/lcd-font

**Comment by Hiradur** [comment]
> Replacement for digital-7.mono.ttf?
> https://github.com/ctrlcctrlv/lcd-font

Looks like 7-segment characters are not finished (see TO DO) so it's not a perfect replacement. Otherwise I'd say yes, they don't seem to be many free/libre digital fonts.
